include/concurrencpp/runtime/runtime.h

```diff
@@ -17,13 +18,13 @@ namespace concurrencpp::details {
         std::vector<std::shared_ptr<executor>> m_executors;
 
        public:
-        void register_executor(std::shared_ptr<executor> executor);
+        CRCPP_API void register_executor(std::shared_ptr<executor> executor);
```

Comment:
wouldn't it make more sense just to mark `executor_collection` as `CRPP_API`?

Reply:
Yes. Originally I was exporting individual functions, but that can't handle polymorphism (namely the executor class hierarchy), so I just switched to exporting whole classes. I just missed this case.